lib/reverseanalyzer.h

```diff
@@ -25,7 +25,7 @@ class Token;
 template<class T>
 class ValuePtr;
 
-void valueFlowGenericReverse(Token* start, const ValuePtr<Analyzer>& a, const Settings& settings);
-void valueFlowGenericReverse(Token* start, const Token* end, const ValuePtr<Analyzer>& a, const Settings& settings);
+void valueFlowGenericReverse(const Token* start, const ValuePtr<Analyzer>& a, const Settings& settings);
+void valueFlowGenericReverse(const Token* start, const Token* end, const ValuePtr<Analyzer>& a, const Settings& settings);
```

Comment:
These should be non-const as we will pass it to analyzer->update, which uses a non-const pointer.